internal/publish/publish.go

```diff
@@ -322,7 +322,13 @@ func (h *PublishHandler) process(ctx context.Context, data *verifyapi.Publish, b
 			metric: "publish-transform-fail", count: 1}
 	}
 
-	n, err := h.database.InsertAndReviseExposures(ctx, exposures, token, !appConfig.BypassRevisionToken)
+	n, err := h.database.InsertAndReviseExposures(ctx, &database.InsertAndReviseExposuresRequest{
+		Incoming: exposures,
```

Comment:
something is missing here - the new revision token will still merge the old revision token (if there is one) and all of the exposures presented.

Thus giving the user a revision token that would let them later revise the keys.

We need to only issue the revision token for keys they successfully inserted.

Reply:
Okay, this was a bit more involved that I originally thought, because I had to change the return type, but this is done now.